.yarnrc.yml

```diff
@@ -0,0 +1,2 @@
+yarnPath: .yarn/releases/yarn-3.1.1.cjs
```

Comment:
[Yarn docs instruct us](https://yarnpkg.com/getting-started/migration#step-by-step) to commit its cjs module, not a mistake.